### DISCUSSION TITLE
Resolution problem on Raspberry 4

### DISCUSSION THREAD

**Discussion by reinhardkrakow** [raysan5/raylib#2435] [Q&A]
Hi,
I use a Raspberry Pi with a 5 inch screen. The resolution of the screen is 800x480.
When I try to draw a circle, the screen displays a flat circle.

As you can see the program detect a screen 800x600 but the screen resolution is 800x480.
[log.txt](https://github.com/raysan5/raylib/files/8501922/log.txt)
[source.txt](https://github.com/raysan5/raylib/files/8501929/source.txt)

Could you please tell me what is wrong.

Kind regards,
Reinhard

**Comment by reinhardkrakow** [comment]
If I use a HDMI screen (1920x1080), the program displays a correct circle.

Reinhard

**Comment by raysan5** [comment]
This could be an issure with raylib detecting display resolution properly

**Comment by reinhardkrakow** [comment]
I was using OpenVG in the past. On the Raspberry Pi 4, this library is not supported anymore. On the previous version RPi 2 3 it was working fine. I try to find a replacement solution with Raylib.

**Comment by reinhardkrakow** [comment]
Hi,
I have another screen 800x480 (4inch). this screen works vertically (width 480, height 800)
I change my program in the initialization by putting:

    const int screenWidth = 480;
    const int screenHeight = 800;

The program recognize now the screen:

INFO: DISPLAY: No graphic card set, trying platform-gpu-card
INFO: DISPLAY: Selected DRM connector mode 480x800 (480x800p@60)
INFO: DISPLAY: Device initialized successfully
INFO:     > Display size: 480 x 800
INFO:     > Screen size:  480 x 800
INFO:     > Render size:  480 x 800

and work perfectly.
I do not know why?

**Comment by reinhardkrakow** [comment]
How I can configure my first screen (5inh) to work like the 4inch with

const int screenWidth = 480;
const int screenHeight = 800;

BR
Reinhard

**Comment by reinhardkrakow** [comment]
Hi,
The nearest display format in raylib for Raspberry Pi 5inch display is 720x480.
I just add in the /boot/config.txt fife :
overscan_right=80 to reduce the width of the display from 800 to 720
